jwql/shared_tasks/shared_tasks.py

```diff
@@ -834,7 +830,6 @@ def run_parallel_pipeline(input_files, in_ext, ext_or_exts, instrument, jump_pip
         for short_name in results:
             try:
                 logging.info("\tWaiting for {} ({})".format(short_name, results[short_name].id))
-                processed_path = results[short_name].get()
```

Comment:
I'm pretty sure we need to keep this. Maybe we can change "processed_path" to "_" if it will make Ruff happier, but I'm pretty sure we need to call the get method. Brian can confirm.